tutorials/dataframe/df106_HiggsToFourLeptons.py

```diff
@@ -193,4 +193,5 @@ def merge_histos(label):
 text.DrawLatex(0.21, 0.80, "#sqrt{s} = 13 TeV, 10 fb^{-1}")
 
 # Save the plot
-c.SaveAs("HiggsToFourLeptons.pdf")
+c.SaveAs("df106_HiggsToFourLeptons.png")
+print("Save figure to df106_HiggsToFourLeptons.png")
```

Comment:
```suggestion
print("Saved figure to df106_HiggsToFourLeptons.png")
```
?

Reply:
Yep, true. I've fixed it for all tutorials in one go!